src/pint/utils.py

```diff
@@ -1366,6 +1366,86 @@ def wavex_setup(model, T_span, freqs=None, n_freqs=None):
     return model.components["WaveX"].get_indices()
 
 
+def dmwavex_setup(model, T_span, freqs=None, n_freqs=None):
+    """
+    Set-up a DMWaveX model based on either an array of user-provided frequencies or the wave number
+    frequency calculation. Sine and Cosine amplitudes are initially set to zero
+
+    User specifies T_span and either freqs or n_freqs. This function assumes that the timing model does not already
+    have any WaveX components. See add_dmwavex_component() or add_dmwavex_components() to add components
```

Comment:
"any WaveX components": should be "DMWaveX"?

Reply:
changed.